dev/lib/product_taxonomy/models/integration_version.rb

```diff
@@ -54,25 +61,48 @@ def load_from_source(integration_path:)
           full_names_by_id:,
           from_shopify_mappings:,
           to_shopify_mappings:,
+          current_shopify_version:,
+        )
+      end
+
+      def generate_all_mappings_file(mappings:, version:, output_path:)
+        File.write(
+          File.expand_path("en/integrations/all_mappings.json", output_path),
+          JSON.pretty_generate(to_json(mappings:, version:)),
         )
       end
+
+      def to_json(version:, mappings:)
+        {
+          version:,
+          mappings: mappings.is_a?(Array) ? mappings : [mappings],
```

Comment:
Nit: The array constructor does the same thing:
```suggestion
          mappings: Array(mappings),
```

Reply:
Not _strictly_ speaking, but practically for what I think is happening here.

Reply:
`Array(mappings)` actually doesn't behave as desired here, it coverts a hash input to an array (rather than wrapping it).

I still simplified this though by having the callers wrap the argument in an array 👍